src/rev_ai/streamingclient.py

```diff
@@ -73,6 +73,9 @@ def start(self, generator, metadata=None):
             + '&content_type={}'.format(self.config.get_content_type_string()) \
             + '&user_agent={}'.format(quote('RevAi-PythonSDK/{}'.format(__version__), safe=''))
 
+        if custom_vocabulary_id:
+            url += '&custom_vocabulary_id={}'.format(quote(custom_vocabulary_id), safe='')
```

Comment:
urlencode is the right way to create a URL. See https://github.com/revdotcom/revai-python-sdk/pull/32/files

Reply:
Josh, I asked Kyle about the status of your PR. I don't recall the status. Since we are modifying this directly, it makes sense to make these changes Kostas